R/selections.R

```diff
@@ -199,6 +199,17 @@ recipes_eval_select <- function(quos, data, info, ..., allow_rename = FALSE,
 
   expr <- expr(c(!!!quos))
 
+  if ((!allow_rename) && any(names(expr) != "")) {
+    offenders <- names(expr)
+    offenders <- offenders[offenders != ""]
+
+    cli::cli_abort(
+      "The following argument{?s} {?was/were} specified but not available: \\
+      {.arg {offenders}}.", 
```

Comment:
```suggestion
      "The following argument{?s} {?was/were} specified but do not exist: \\
      {.arg {offenders}}.", 
```

I tripped up a bit on "available" here... seems to hint to me that, like, a package needs to be loaded or some other condition needs to be met to use the argument rather than the argument not existing?